internal/sdkprovider/service/kafkatopic/kafka_topic.go

```diff
@@ -308,18 +318,22 @@ func resourceKafkaTopicCreate(ctx context.Context, d *schema.ResourceData, m int
 	topicName := d.Get("topic_name").(string)
 	partitions := d.Get("partitions").(int)
 	replication := d.Get("replication").(int)
+	topicDescription := d.Get("topic_description").(string)
+	ownerUserGroupID := d.Get("owner_user_group_id").(string)
 
 	config, err := getKafkaTopicConfig(d)
 	if err != nil {
 		return diag.Errorf("config to json error: %s", err)
 	}
 
 	createRequest := aiven.CreateKafkaTopicRequest{
-		Partitions:  &partitions,
-		Replication: &replication,
-		TopicName:   topicName,
-		Config:      config,
-		Tags:        getTags(d),
+		Partitions:       &partitions,
+		Replication:      &replication,
+		TopicName:        topicName,
+		Config:           config,
+		Tags:             getTags(d),
+		TopicDescription: &topicDescription,
+		OwnerUserGroupId: &ownerUserGroupID,
```

Comment:
This is will send empty strings when there is no value set.

Reply:
I can fix this later, because I see similar issues there, which are not related to the PR.

Reply:
I think i need to fix this to get acceptance tests pass, but i'll leave the other occasion to you then 👍🏼